vmdk_plugin/utils/volumeProperties/VolumePropertiesUtil.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
```

Comment:
need to rename this file name something like VolumePropertiesVerificationUtil.go certainly not VolumePropertiesUtil.go